uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/api/v3/ContactTest.php

```diff
@@ -45,7 +45,7 @@ public function testGetLeaveManageesDoesNotReturnDeceasedORDeletedContacts() {
     $this->setContactAsLeaveApproverOf($this->manager, $contact3, null, null, true, 'has leaves approved by');
     $this->setContactAsLeaveApproverOf($this->manager, $contact4, null, null, true, 'has things managed by');
 
-    $result = civicrm_api3('Contact', 'getleavemanagees', ['managed_by' => $this->manager['id']]);
+    $result = $this->callGetLeaveManagees(['managed_by' => $this->manager['id']]);
```

Comment:
Why not use check_permissions=true for all the tests? In fact, I was just thinking we can have a BaseAPITestCase class, which would serve as base class to all the tests under tests/phpunit/api/v3. This class would:
- automatically give the 'Access Ajax API' permission to the current logged in user on its setup method
- have a helper method to call civicrm_api3 passing check_permission to true

The idea is to avoid having these ACL/permissions issues again and again. 

We don't need to refactor all the other tests now, but having it for this class would be a good start.